low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1224,7 +1380,20 @@ def _parse_voltage_info(self, index, date, _time, sensor, sensor_num, event, sta
 #            "description": event
 #        }
 #
+#        if (threshold.lower() in ['low', 'high'] and status.lower() == "asserted"):
+#            self.faulty_resources[sensor_name] = {
+#                'status': status,
+#                'event': event,
+#                'device_id': sensor,
+#                'fru_type': self.TYPE_VOLTAGE
+#            }
+#        elif (alert_type in ['fault_resolved', 'miscellaneous'] or \
+#            (threshold.lower() in ['low', 'high'] and status.lower() == "deasserted")) \
+#                and sensor_name in self.faulty_resources:
+#            del self.faulty_resources[sensor_name]
+#
```

Comment:
Pls remove these lines if not required.

Reply:
We had disabled/commented on this code because this sensor is currently not available in the super micro server. once it is available we need to enable/uncomment this code.
So we required this code.

Reply:
yes we want to main this code